packages/react-instantsearch-core/src/lib/use.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+type Use = <T>(promise: Promise<T>) => T;
+const useKey = 'use' as keyof typeof React;
+
+// @TODO: Remove this file and import directly from React when available.
+export const use = React[useKey] as Use;
```

Comment:
why is this not available yet now?

Reply:
Cause `use` is still a canary feature therefore it's not available in typings